spec/models/topic_spec.rb

```diff
@@ -0,0 +1,71 @@
+# == Schema Information
+#
+# Table name: topics
+#
+#  id         :bigint           not null, primary key
+#  label      :string
+#  icon       :string
+#  created_at :datetime         not null
+#  updated_at :datetime         not null
+#
+require 'rails_helper'
+
+# RSpec.describe Topic, type: :model do
+#   pending "add some examples to (or delete) #{__FILE__}"
+# end
+
+RSpec.describe Api::V1::TopicsController, type: :controller do
```

Comment:
this looks good, but you are testing the requests again, so I would suggest to move this tests to the `spec/requests/api/v1/topics_spec.rb` file, but here on this file to test model only behaviours, like the validations